src/SourceBuild/patches/command-line-api/0002-Use-net8.0-TFM-when-building-with-source-build.patch

```diff
@@ -0,0 +1,68 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Matt Thalman <mthalman@microsoft.com>
+Date: Thu, 6 Apr 2023 09:16:13 -0500
+Subject: [PATCH] Use net8.0 TFM when building with source-build
+
+Backport: https://github.com/dotnet/command-line-api/pull/2150
+---
+ Directory.Build.props                                       | 6 +++---
+ .../System.CommandLine.Benchmarks.csproj                    | 4 ++--
```

Comment:
Yes, exactly. I didn't patch any of the test projects nor the `System.CommandLine.Suggest` project since none of those are targeted by source-build. The patch also differs (in context) from the changes in https://github.com/dotnet/command-line-api/pull/2150 because it is based on a much older commit of the command-line-api repo that is referenced in the VMR.